lib/twilio-ruby/rest/client.rb

```diff
@@ -71,6 +72,15 @@ def initialize(username=nil, password=nil, account_sid=nil, region=nil, http_cli
       # Makes a request to the Twilio API using the configured http client
       # Authentication information is automatically added if none is provided
       def request(host, port, method, uri, params={}, data={}, headers={}, auth=nil, timeout=nil)
+        whitelisted_uris = ["Messages.json", "Calls.json"]
+        whitelisted_numbers = ENV.fetch("WHITELISTED_NUMBERS", "").split
```

Comment:
Have you considered making use of the config object to pass these numbers in from the outside? For example, in `config/initializers/twilio.rb`:
`Twilio.configure do |config|
  config.account_sid = ENV.fetch("TWILIO_SID")
  config.auth_token  = ENV.fetch("TWILIO_TOKEN")
  config.whitelisted_numbers = ENV.fetch("WHITELISTED_NUMBERS", "").split
end`

If we do it like this maybe we can create a merge request against the Twilio gem, this seems to me like a feature more people would want, it might get approved.

Reply:
Good suggestion, I will move this to the config section 👌